Vagrantfile

```diff
@@ -5,31 +5,31 @@
 # configures the configuration version (we support older styles for
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
-Vagrant.configure("2") do |config|
+Vagrant.configure('2') do |config|
   # The most common configuration options are documented and commented below.
   # For a complete reference, please see the online documentation at
   # https://docs.vagrantup.com.
 
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
-  config.vm.box = "ubuntu/xenial64"
+  config.vm.box = 'ubuntu/xenial64'
 
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine and only allow access
   # via 127.0.0.1 to disable public access
-  config.vm.network "forwarded_port", guest: 7474, host: 7474, host_ip: "127.0.0.1"
-  config.vm.network "forwarded_port", guest: 7473, host: 7473, host_ip: "127.0.0.1"
-  config.vm.network "forwarded_port", guest: 7472, host: 7472, host_ip: "127.0.0.1"
+  config.vm.network 'forwarded_port', guest: 7474, host: 9474, host_ip: '127.0.0.1'
+  config.vm.network 'forwarded_port', guest: 7473, host: 9473, host_ip: '127.0.0.1'
+  config.vm.network 'forwarded_port', guest: 7472, host: 9472, host_ip: '127.0.0.1'
```

Comment:
I should not have committed this host port change

Reply:
Cool, I just put them back